lib/ui/src/index.js

```diff
@@ -16,6 +17,13 @@ HelmetProvider.displayName = 'HelmetProvider';
 
 const Container = process.env.XSTORYBOOK_EXAMPLE_APP ? React.StrictMode : React.Fragment;
 
+const base =
+  document.location.protocol === 'file:'
+    ? `${document.location.origin}${document.location.pathname}`
+    : '/';
+
+console.log({ base });
```

Comment:
This looks like it's not needed (all the changes in this file?)

Reply:
will remove ✅